app/views/global_activities/index.html.erb

```diff
@@ -28,5 +28,8 @@
   </div>
 </div>
 
+<script>
+  var gaUrlQueryParams= <%= raw @filters.to_json %> 
```

Comment:
Is there any vulnerability here? We are white-labeling the `activity_filters`, but the values can still be anything, or is that filtered out? Just double-checking, need to be careful when using `raw`.

Reply:
It there is will be wrong format, back end will fail first one, because we except, that only we will generate this params. We need separate task to implement also back end checking. On front end if there is will be wrong format, it simply not set correct filter, but will continue working.